iocage/cli/activate.py

```diff
@@ -23,20 +23,40 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """activate module for the cli."""
 import click
+import libzfs
 
-import iocage.lib.ioc_common as ioc_common
-import iocage.lib.iocage as ioc
+import iocage.lib.Datasets
+import iocage.lib.Logger
 
 __rootcmd__ = True
 
 
 @click.command(name="activate", help="Set a zpool active for iocage usage.")
 @click.argument("zpool")
-def cli(zpool):
+@click.option("--log-level", "-d", default=None)
+@click.option("--mountpoint", "-m", default="/iocage")
+def cli(zpool, log_level, mountpoint):
     """Calls ZFS set to change the property org.freebsd.ioc:active to yes."""
-    ioc.IOCage(activate=True).activate(zpool)
+    logger = iocage.lib.Logger.Logger(print_level=log_level)
+    zfs = libzfs.ZFS(history=True, history_prefix="<iocage>")
```

Comment:
The helpers.py file is meant to be used by the library only. We can create a public interface that both the helpers and cli can create a zfs instance from. Do you think that's necessary here? Basically we want to get a list of available ZFS pools on the system here.